code/modules/jobs/job_types/brigphys.dm

```diff
@@ -37,6 +37,9 @@
 	l_hand = /obj/item/storage/firstaid/medical
 	head = /obj/item/clothing/head/soft/sec/brig_phys
 
+	implants = list(/obj/item/implant/mindshield)
```

Comment:
unintended change

Reply:
Not sure I follow?

Reply:
you are adding mindshields to brig physician

Reply:
Ah I fucked up that merge resolution